core/src/utils/slot-mutation-controller.ts

```diff
@@ -50,6 +50,18 @@ export const createSlotMutationController = (
 
     hostMutationObserver.observe(el, {
       childList: true,
+      /**
+       * This fixes an issue with the `ion-input` component
+       * not re-rendering in some cases when using the
+       * label slot functionality.
```

Comment:
```suggestion
       * This fixes an issue with the `ion-input` and
       * `ion-textarea` not re-rendering in some cases
       * when using the label slot functionality.
```

Tiny nitpick.

Reply:
For my own understanding, where/how does `ion-textarea` come into play here? TBH, I just lifted the commit message from https://github.com/ionic-team/ionic-framework/pull/28995/commits/13042029c96a7564d37a7d75ca3cdb6ae6026246. I'm not opposed to this change, just wanna double check that it is/was an issue we were seeing.